assets/src/edit-story/app/media/categories/actions.js

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Shouldn't this file be under media/media3p instead of media/ ?

Reply:
Done.